README.md

```diff
@@ -1,5 +1,6 @@
 # Custom attributes for Home Assistant
 
+[![hacs_badge](https://img.shields.io/badge/HACS-Custom-41BDF5.svg?style=for-the-badge)](https://github.com/hacs/integration)
```

Comment:
It will look nicer if it has the same styles as the rest of badges:

```
[![hacs_badge](https://img.shields.io/badge/HACS-Custom-41BDF5.svg?style=flat-square)](https://github.com/hacs/integration)
```